src/components/ZoomHandler.component.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { select, event, zoom as d3ZoomFactory, ZoomBehavior } from 'd3';
```

Comment:
event is not exported anymore.
It was used as singleton in event handler.
now in the migration guide we see they have fixed that shitty behavior and do a real event handler